gitconfig

```diff
@@ -25,3 +25,5 @@
   autosquash = true
 [include]
   path = ~/.gitconfig.local
+[diff]
+	colorMoved = zebra
```

Comment:
Should we match the indentation here?

Reply:
Yeah, that was added when I copy-pasted from the git auto-addition. Nice catch!

Reply:
Hrm, this was weird, it didn't look over-indented when I checked back. Replaced the spaces manually in case there were invisible characters there.